src/app/components/LiteSiteCta/liteSiteConfig.tsx

```diff
@@ -0,0 +1,10 @@
+import { Services } from '#app/models/types/global';
+
+export const defaultTranslations = {
+  onboardingMessage: `You’re viewing a text-only version of this website that uses less data. View the main version of the website including all images and videos.`,
```

Comment:
Lets make sure we get translations/copy that doesn't refer to this as a  text-only version

Reply:
These are fallback translations so in theory should never been seen by users. We have added actual translations into the Gahuza config. These fallbacks are useful for Storybook: https://wsteam1-1349-lite-site-cta--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&id=components-litesitecta--component&viewMode=story